build/cqs.tsv

```diff
@@ -192,6 +192,7 @@ org.geotools.xsd:gt-xsd-wfs	15.1	compile
 org.hsqldb:hsqldb	2.3.0	compile
 org.jamon:jamon-runtime	2.4.1	compile
 org.jdom:jdom	1.1.3	compile
+org.jgrapht:jgrapht-core	1.0.1	compile
```

Comment:
Looks like some work was done to remove the problematic files but not sure it's hosted anywhere, or got re-merged back to the main branch: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=11214